app/controllers/idv/review_controller.rb

```diff
@@ -68,5 +76,13 @@ def submit_applicant
       idv_attempter.increment
       resolution
     end
+
+    def password
```

Comment:
super minor detail, but I usually put methods in order of dependencies (so `password` below where it is used, which here is `valid_password?`). I find this makes diffs more readable because you read from top to bottom and see why each thing is used.